README.md

```diff
@@ -1,4 +1,4 @@
-# image-inspector-client
+# image-inspector-client-ruby
```

Comment:
I think the title need not reflect the repo name (github already shows that), the gem name is more interesting to the reader.  Perhaps:
# image-inspector-client Ruby Gem

Reply:
hmm, that was serious bikeshedding on my part :bike: :house:, doesn't matter.

> *What’s in a name? That which we call image-inspector-client
> By any other name would Look as Good To Me :+1:;*

Reply:
I think keeping `image-inspector-client` is a good idea now that you mention it (due to repo name!=gem name). Does that work for you?

Reply:
Sure.